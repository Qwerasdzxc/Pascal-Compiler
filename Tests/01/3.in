291.483 827.554

