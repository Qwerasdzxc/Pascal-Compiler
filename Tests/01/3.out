1119.04 -536.07 0.35